content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -8,7 +8,37 @@ menu:
     parent: About the project
 ---
 
+## v1.9.3 [2022-02-02]
+
+{{% note %}} To address a bug that impacted communication between the browser’s main thread and background workers, we bumped the 1.9.2 release version to 1.9.3.
+{{% /note %}}
+
+### Features
+- Add ability to rename TICKscripts.
+- Add the following enhancements to the `InfluxDB Admin - Queries` tab:
+  - `CSV download` button. 
+  - Rename `Running` column to `Duration`.
+  - Add `Status` column. When hovering over the `Duration` column, status shows `Kill` confirmation button.
+  - Modify the `CSV` export to include the `Status` column.
+- Upgrade to use new `google.golang protobuf` library.
+
+### Bug Fixes
+- Ability to log the InfluxDB instance URL when a ping fails, making connection issues easier to identify.
+- Repair enforcement of one organization between multiple tabs.
+- Configure HTTP proxy from environment variables in HTTP clients. Improvements were made to:
+  - Token command within `chronoctl` 
+  - OAuth client
+  - Kapacitor client 
+  - Flux client
+
+
+#### Security
+- Upgrade `github.com/microcosm-cc/bluemonday` to resolve CVE-2021-42576.
+
 ## v1.9.1 [2021-10-08]
+{{% warn %}} **Warning:** Chronograf 1.9.1 was released with a bug which impacted communication between the browser's main thread 
```

Comment:
1.9.2....  not 1.9.1.  We effectively did not create 1.9.2 as a result. 

I believe that we have another example of this.  But, I think we should tag v1.9.2 [unreleased]  and then this comment.

Reply:
I will reword for clarity